float64 probability
string object_name
int16 id
float64 x
float64 y
